jetty9-base/src/main/jetty-base/etc/java-util-logging.properties

```diff
@@ -0,0 +1,9 @@
+.level = INFO
+io.netty.level = INFO
+io.gcr.level = INFO
```

Comment:
Currently the approach taken by the gcloud-java client library is to have fixed list of known log names that it explicitly scans for and removes the stackdriver handler from.

This is the approach that should also stop looping, but it not.

The same list could simply be applied within the log handler and any logs from those handlers would be discarded.   This list could either be a complete list, in which case we would not need either the threadlocal approach nor the handler deregistration.

Alternately the list can just be limited to the loggers that are not reacting to the logging properties configuration.

Currently I don't like any solution.      My preference would be to have no filtering and just loop protection.   Overly verbose logging should be handled gracefully (with discarded logs and a message to that effect) and then the user can just configure logging to only log what they are interested in.

So I need to investigate why those netty classes are ignoring the properties file.